network-configuration/advanced-network-configuration/src/main/java/MemberB.java

```diff
@@ -0,0 +1,37 @@
+import com.hazelcast.config.Config;
+import com.hazelcast.config.SSLConfig;
+import com.hazelcast.config.ServerSocketEndpointConfig;
+import com.hazelcast.core.Hazelcast;
+import com.hazelcast.core.HazelcastInstance;
+
+public class MemberB {
+
+    public static void main(String[] args) {
+        Config config = new Config();
+
+        config.getAdvancedNetworkConfig().setEnabled(true);
+        // advanced network configuration creates by default a member server socket on port 5701
+        // in this example we only configure explicitly the WAN server socket endpoint
+        config.getAdvancedNetworkConfig().addWanEndpointConfig(createWanEndpointConfig());
+
+        HazelcastInstance hz = Hazelcast.newHazelcastInstance(config);
+    }
+
+    // This is the the passive side of WAN replication. A server socket is created that listens
+    // for incoming connections. SSL configuration needs to be the same as active side to allow
+    // connection establishment.
+    private static ServerSocketEndpointConfig createWanEndpointConfig() {
+        ServerSocketEndpointConfig endpointConfig = new ServerSocketEndpointConfig();
+        endpointConfig.setName("active-wan")
+                      .setSSLConfig(new SSLConfig()
+                              .setEnabled(true)
+                              .setProperty("trustStore", "truststore")
```

Comment:
I think `keyStore` and `keyStorePassword` should be used here for replication ClusterA -> ClusterB (or both sides can use both keystore and truststore if mutual authentication is enabled)